combinations/hash.tsv

```diff
@@ -472,3 +472,4 @@ macs2=2.2.9.1,bedtools=2.31.0,ucsc-bedclip=377,ucsc-bedgraphtobigwig=445
 bioconductor-flowsom=1.18.0,mscorefonts=0.0.1
 bioconductor-flowsom=1.16.0,mscorefonts=0.0.1
 bioconductor-flowai=1.2.9,bioconductor-flowcore=1.42.0,mscorefonts=0.0.1
+python=3.9,tensorflow=2.10.0,scikit-learn=1.1.2,umap-learn=0.5.3,matplotlib=3.6.0
```

Comment:
you can remove python here, it will come in implicitly

Reply:
I have removed python, thank you